SETUP/tests/manual_web/split_test/switchable_split.php

```diff
@@ -0,0 +1,36 @@
+<?php
+$relPath="../../../../pinc/";
+include_once($relPath.'base.inc');
+include_once($relPath.'slim_header.inc');
+
+// Demonstrate two pane vertical split
+
+$title = _('Two Pane Switchable Split with Flex Layout');
+
+$js_files = [
+    "$code_url/scripts/splitControl.js",
+    "switchable_split.js",
+];
+
+$header_args = [
+    "js_files" => $js_files,
```

Comment:
What you have here is fine. Note also you can do:
```
$header_args = [
    "js_files" => [
        "$code_url/scripts/splitControl.js",
        "switchable_split.js",
    ],
    "css_files" => ["split_test.css"],
    "body_attributes" => "style='margin: 0; overflow: hidden;'",
];
```

Reply:
Changed all files to be like this.